code/src/java/pcgen/rules/context/RuntimeReferenceContext.java

```diff
@@ -36,7 +36,7 @@
 import pcgen.cdom.reference.ManufacturableFactory;
 import pcgen.cdom.reference.ReferenceManufacturer;
 import pcgen.cdom.reference.SimpleReferenceManufacturer;
-import pcgen.persistence.PersistenceLayerException;
+// import pcgen.persistence.PersistenceLayerException;
```

Comment:
why commented out?

Reply:
One of the tests flagged it as unused. I didn't want to just delete it. so I commented it.

Reply:
It's safe to remove :-)